knowledge-base/adr/0018-openapi-generator.md

```diff
@@ -0,0 +1,45 @@
+# OpenApi Client generator
+
+## Status
+
+proposed
```

Comment:
If we merge this now should we go to `accepted`?

Reply:
Good point. Maybe we should also write some more things down. I will do this on a separate PR.